docs/src/install.md

```diff
@@ -0,0 +1,23 @@
+# [Installation](@id doc: Installation)
+
+## QuantumToolbox.jl
```

Comment:
Maybe also make this as sub-sub-title (`###`) to avoid unnecessary expansion in sidebar.

Reply:
I think that we can move the subtitle `## Introdution`` under the introduction, before the first example. Such that we ha a single page, with every information inside. Indeed, most of the people will read only the first page, and so this page is very important

Reply:
Let us discuss this in `zulip`?